client/src/QuestionList.tsx

```diff
@@ -0,0 +1,88 @@
+import * as React from 'react';
+import Typography from '@material-ui/core/Typography';
+import { makeStyles } from '@material-ui/core/styles';
+import type { Theme } from '@material-ui/core/styles';
+
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemText from '@material-ui/core/ListItemText';
+import Collapse from '@material-ui/core/Collapse';
+import ExpandLess from '@material-ui/icons/ExpandLess';
+import ExpandMore from '@material-ui/icons/ExpandMore';
+
+const useStyles = makeStyles<Theme, QuestionListProps> ({
+    questionList: {
+        width: '100%',
+    },
+});
+
+type QuestionListProps = {
+    questionList: { question: string, answer: string }[],
+};
+
+function QuestionList(props: QuestionListProps) {
```

Comment:
aaaayee bummer
sorry, i should've caught this in the designs earlier and said something to the team
this should use the same component as the homepage
we wouldn't want 2 accordion-esque components around, would make our page look inconsistent
checkout https://github.com/Nonprofit-Exchange-Hub/web-app/blob/9e0f96dfcb7bc0031e5750523b47f5af46f2943e/client/src/Home.tsx#L194

Reply:
No worries! It was good for me to figure it out, but it's easy to change.